runtime/lua/vim/lsp/util.lua

```diff
@@ -1258,7 +1258,7 @@ do
   --- </pre>
   function M.buf_diagnostics_signs(bufnr, diagnostics)
     for _, diagnostic in ipairs(diagnostics) do
-      vim.fn.sign_place(0, sign_ns, diagnostic_severity_map[diagnostic.severity], bufnr, {lnum=(diagnostic.range.start.line+1)})
+      vim.fn.sign_place(0, sign_ns, diagnostic_severity_map[diagnostic.severity], bufnr, {lnum=(diagnostic.range.start.line+1),priority=90})
```

Comment:
`90` is pretty arbitrary, I think this should be a configurable value, idn't it?

Reply:
Yes, so this is a draft. The maintainer is required to determine the name of the variable